rasa_sdk/knowledge_base/actions.py

```diff
@@ -213,14 +225,14 @@ async def _query_attribute(
 
         Returns: list of slots
         """
-
         object_name = get_object_name(
             tracker,
+            object_type,
             self.knowledge_base.ordinal_mention_mapping,
             self.use_last_object_mention,
         )
 
-        if object_name is None or not attribute:
+        if not object_name or not attribute:
```

Comment:
It is actually more explicit to retain the original if check `if object_name is None`

Reply:
Done 👍

Reply:
This change was not actually made, please revert 😄

Reply:
Oops 😅  Reverted it